ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/MdbDeliveryDependenciesProcessor.java

```diff
@@ -78,15 +78,14 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
                     if (mdbDescription.isClusteredSingleton()) {
```

Comment:
Any reason the above `addDependency` at line 76 (and another one at line 98-99) is not replaced?

Reply:
Yes, other addDependency() methods have different Java method signature @chengfang